scripts/cangen/README.md

```diff
@@ -0,0 +1,41 @@
+# Cangen
+Cangen is a cli package used to generate C++/C files from a corresponding dbc file. This tool automates the can generation process and ensures can generation is consistent between all projects. 
```

Comment:
CLI, DBC, CAN should all be capitalized. Since we generate C++, you can remove the "/C"